app/src/components/SidePanel.tsx

```diff
@@ -8,7 +8,9 @@ import { Options } from "./Options"
 import SearchBar from "./SearchBar"
 import { Spacer } from "./Spacer"
 
-const SidePanelRoot = styled.aside``
+const SidePanelRoot = styled.aside`
+  overflow-y: auto;
+`
```

Comment:
This introduces an issue when the sidebar is overflowing, a horizontal scrollbar is shown